### DISCUSSION TITLE
Bloated UI & Videos for beginner?

### DISCUSSION THREAD

**Discussion by luwol03** [inventree/InvenTree#3248] [General]
Hey,

I discovered inventree a few months ago, but only saved to for later. Now I'm actually searching for a storage place coordination and planning software and hope I found the right with inventree.

After a first try via the demo page I thought this UI is very bloated and not intuitive enough to use. What do you think?

I also want to ask if there are some videos for starters which show the basics functionality and present inventree so more and more people have a easier starting.

luwol03

**Comment by matmair** [comment]
Hi there @luwol03 
Thanks for the encouragement and suggestions ;-) !

UI:
Jokes aside we are aware that the Interface is a bit 'technical'. The core team are only a handful of ppl. and we are all engineers by training so the UI is a lot more of learning-by-doing and less be-and-end-all craftsmenship. If you have experience in this field (or not) we are always open for constructive suggestions or PRs. UI and UX is hard - we are certainly open for improvements.
You can turn of a lot of the functions in the settings, that makes it less dense. Maybe try a less-privileged user like reader to have a look at a more basic setup.
The software supports small-scale production and storage of complicated electromechanical assemblies so there is a lot of stuff that needs a place.

Videos:
As we are a small team we sadly do not have the capacity to provide videos. There is a extensive documentation site on https://inventree.org .
I encourage you to look there (or use google - the docs are indexed). If you are not able to find answers there we have a small Reddit community or - better jet - GitHub Issues and discussions are open (as you already know).

Simpler solutions:
On a more personal note: If your goal is to manage you personal belongings the project might be overkill and something like snipe-it or grocy could be an easier and user-friendlier route.

I would love to have a new community member (or constructive watcher) on board and wish you best regards from sunny Austria
Matthias

**Reply by luwol03** [reply]
Hey Matthias,

thank you for your quick replay, never found an open source project where I got an answer faster than today and also personally from the owner.

I really like the software because there are so much options, but I really need to get an overview of each functionality first. And this could not be archived by just playing with the demo. Thank you for mentioning the docs, I have to read them like a manual for using the software. I will ask again, if there is something unclear.

Thank you for also suggesting other open source solutions, this is not normal for projects, but shows the attitude of the project. Awesome.

Other alternatives:
No, they are not complex enough. I have a view storage places and a huge amount of electronical components which I want to sort and manage with such a system.

Open questions:
1. one thing I asked me is, how do I print labels on normal paper in a grid of specific size so that I can just cut the A4 paper to e.g. 5x20 labels for 100 little boxes from an electronical magazine?
![](https://www.udobaer.de/INTERSHOP/static/WFS/UDOBAER-UBDE-Site/-/UDOBAER/de_DE/L/i0000170685.jpg)

2. how can I just decrease the in stock amount just by one if I take a part from stock, very easily?

3. How can I auto generate a IPN from a given schema for each new part?

4. Is it also possible to track some not consumable material like tools (e.g. Wrenches). I want to say I have Wrenches in following sizes and if I want to borrow the wrench to somebody in a specific size I want to also mark that as currently in borrow and if I get it I can easily unmark it as borrowed. And it would be also cool to see which items are currently borrowed.

Best regards
luwol03

**Reply by matmair** [reply]
Hi there @luwol03 
I am one of the core devs, the maintainer is @SchrodingersGat . InvenTree provides a lot of functionality so I would encourage reading through the docs.

1. That is currently not implemented; Could be a label plugin.
2. You can go to the stock item page (for eg. https://demo.inventree.org/stock/item/408/ ) and use the actions there to remove stock 
![grafik](https://user-images.githubusercontent.com/66015116/175791720-bce611a1-e26d-4f38-afcd-d721283b0829.png)
3. You can make sure that your entered IPN matches a regex - Auto IPN is not a function we have - there is a PR for that #2848
4. You can assign stock items to users after enabling ownership controls in the global settings

**Reply by luwol03** [reply]
Hey,

I read the documentation yesterday like a book. Very awesome work the docs. Now it's much clear how everything works. I plan to set a first instance up and play with it from the beginning for better understanding.

1. I've not found something about multiple labels at once in a plug-in. Does this actually work? Or do I have to make some kind of queue myself?
3. there is already a Pull request? From what I read in the open issue, there is no direct option to hooking in the UI with a plug-in to make an auto generate button. The only workaround currently is with a pre save hook of parts and then set the IPN equal to the internal ID.

**Reply by matmair** [reply]
1. You can print multiple labels at once from tables.
2. Sorry I misphrased that. There is an issue with dicsussions how it could be resolved. Once there is movement on that I think there will be a PR implementing it. UI integration is certainly not off the books if somebody implements it.

**Reply by luwol03** [reply]
Ok. Thank you for your time answering my beginner questions. I will reach you again if I have any further questions while trying inventree. 

Is there actually a Discord Server or some more chat based forum for inventree like other projects have it? Or are discussions the preferred way for questions?

**Reply by matmair** [reply]
Discussions or Issues are the prefered way as they are easier to be found on google.
